spec/features/unauth_users/user_manages_cart_spec.rb

```diff
@@ -30,11 +30,19 @@
   it "allows users to increase the quantity of an item in their cart" do
     create_item_and_add_to_cart
     visit cart_path
-    click_link_or_button "Add Another"
+    page.click_link('', :href => '/cart_items?item=salted-caramel-peanut-butter-cup')
     expect(page).to have_content("Quantity: 2")
     expect(page).to have_content("12.00")
   end
 
+  it "allows user to delete all of one item at once" do
+    create_item_and_add_to_cart
+    visit cart_path
+    page.click_link('', :href => '/cart_items?item=salted-caramel-peanut-butter-cup')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.